sample-apps/react-native/dogfood/src/screens/LiveStream/JoinLiveStream.tsx

```diff
@@ -51,6 +51,12 @@ export const JoinLiveStream = ({
     flexDirection: orientation === 'landscape' ? 'row' : 'column',
   };
 
+  const isValidCallId = () => {
+    // Allows only alphabets, numbers, -(hyphen) and _(underscore)
+    const callIdRegex = /^[A-Za-z0-9_-]*$/g;
+    return callId && callId.match(callIdRegex);
+  };
```

Comment:
```suggestion
  const isValidCallId = callId && callId.match(/^[A-Za-z0-9_-]*$/g);
```

This doesn't need to be a function I think

Reply:
Fixed